docs/user/operation-guide/_index.md

```diff
@@ -165,7 +185,7 @@ mode]({{< relref "../scraping-service" >}}), where breaking up your scrape confi
 multiple Instances is required for sharding and balancing scrape load across a
 cluster of Agents.
 
-## Instance sharing
+## Instance sharing (Beta)
```

Comment:
Is this really beta? If it has been there since 0.5.0?

Reply:
Changed to stable